src/views/SeedPhrase/Congratulations.vue

```diff
@@ -0,0 +1,76 @@
+<template>
```

Comment:
We can put this into `Components` instead and toggle it for a few seconds inside the confirm seed sceen
something like this?
This removes need for route

```
<SetupCompleted v-if="showCongrats" />
...
this.showCongrats = true
this.setTimeout(() => // Unlock wallet, 5000)
```